### DISCUSSION TITLE
Does the reference architecture allow for Elasticsearch to run on the "Private Data Subnet"? (In other words, could the "Private Data Subnet" host, in addition to DBs, Elasticsearch?)

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#319] [Help] [answered]
Does the reference architecture allow for Elasticsearch to run on the "Private Data Subnet"? (In other words, could the "Private Data Subnet" host, in addition to DBs, Elasticsearch?)

**Comment by yorinasub17** [comment, marked as answer]
We currently do not support deploying Elasticsearch into the Reference Architecture in the initial deployment. However, you can use our [elasticsearch](https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/modules/data-stores/elasticsearch) service catalog module to deploy Amazon Opensearch Service (formerly known as Managed Elasticsearch Service) into the Reference Architecture post hand off.
